<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lblFreq1">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>a</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSlider" name="sldFreq1">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="sldFreq2">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblPhase">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Î´</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSlider" name="sldPhase">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblFreq2">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>b</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="PlotWidget" name="lissajousPlot"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>sldFreq1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>update_freq1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldFreq2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>update_freq2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldPhase</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>update_phase()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>update_freq1()</slot>
  <slot>update_freq2()</slot>
  <slot>update_phase()</slot>
 </slots>
</ui>
